src/miniscript/astelem.rs

```diff
@@ -259,11 +259,7 @@ impl<Pk: MiniscriptKey, Ctx: ScriptContext> fmt::Debug for Terminal<Pk, Ctx> {
                 Terminal::After(t) => write!(f, "after({})", t),
                 Terminal::Older(t) => write!(f, "older({})", t),
                 Terminal::Sha256(ref h) => write!(f, "sha256({})", h),
-                Terminal::Hash256(h) => {
-                    let mut x = h.into_inner();
-                    x.reverse();
-                    write!(f, "hash256({})", sha256d::Hash::from_inner(x))
-                }
+                Terminal::Hash256(ref h) => write!(f, "hash256({})", h), // reverse would handled correctly by rust-bitcoin
```

Comment:
This comment is incorrect, right? We are handling the display here in this lib now, right?

Reply:
Yep, this comment is incorrect. Thanks for catching this!